app/scripts/lib/setupSentry.js

```diff
@@ -90,7 +89,7 @@ export default function setupSentry({ release, getState }) {
   }
 
   const environment =
-    METAMASK_BUILD_TYPE === BuildType.main
+    METAMASK_BUILD_TYPE === 'main'
```

Comment:
should we continue to use constants for these strings?

Reply:
We could keep the enum, but there is no guarantee it matches what is defined in `builds.yml`. Would you prefer that?

Reply:
I'll leave that up to you, but in general we don't seem to check it that much after this refactor. Which seems like a more sensible approach anyway

Reply:
```suggestion
    METAMASK_BUILD_TYPE === process.env.METAMASK_BUILD_TYPE_DEFAULT
```